stan/math/prim/prob/loglogistic_lpdf.hpp

```diff
@@ -0,0 +1,148 @@
+ *
+ * @tparam T_y type of scalar.
+ * @tparam T_scale type of scale parameter.
+ * @tparam T_shape type of shape parameter.
+ * @param y (Sequence of) scalar(s).
+ * @param alpha (Sequence of) scale parameter(s)
+ * for the loglogistic distribution.
+ * @param beta (Sequence of) shape parameter(s) for the
+ * loglogistic distribution.
+ * @return The log of the product of the densities.
+ * @throw std::domain_error if any of the inputs are not positive and finite.
+ */
+template <bool propto, typename T_y, typename T_scale, typename T_shape,
+          require_all_not_nonscalar_prim_or_rev_kernel_expression_t<
+              T_y, T_scale, T_shape>* = nullptr>
+return_type_t<T_y, T_scale, T_shape> loglogistic_lpdf(const T_y& y,
+                                                      const T_scale& alpha,
+                                                      const T_shape& beta) {
+  using T_partials_return = partials_return_t<T_y, T_scale, T_shape>;
+  using T_y_ref = ref_type_if_t<!is_constant<T_y>::value, T_y>;
+  using T_scale_ref = ref_type_if_t<!is_constant<T_scale>::value, T_scale>;
+  using T_shape_ref = ref_type_if_t<!is_constant<T_shape>::value, T_shape>;
+  using std::pow;
+  static const char* function = "loglogistic_lpdf";
+  check_consistent_sizes(function, "Random variable", y, "Scale parameter",
+                         alpha, "Shape parameter", beta);
+
+  T_y_ref y_ref = y;
+  T_scale_ref alpha_ref = alpha;
+  T_shape_ref beta_ref = beta;
+
+  decltype(auto) y_val = to_ref(as_value_column_array_or_scalar(y_ref));
+  decltype(auto) alpha_val = to_ref(as_value_column_array_or_scalar(alpha_ref));
+  decltype(auto) beta_val = to_ref(as_value_column_array_or_scalar(beta_ref));
+
+  check_positive_finite(function, "Random variable", y_val);
+  check_positive_finite(function, "Scale parameter", alpha_val);
+  check_positive_finite(function, "Shape parameter", beta_val);
+
+  if (size_zero(y, alpha, beta)) {
+    return 0.0;
+  }
+  if (!include_summand<propto, T_y, T_scale, T_shape>::value) {
+    return 0.0;
+  }
+
+  operands_and_partials<T_y_ref, T_scale_ref, T_shape_ref> ops_partials(
+      y_ref, alpha_ref, beta_ref);
+
+  const auto& inv_alpha
+      = to_ref_if<!is_constant_all<T_y, T_scale>::value>(inv(alpha_val));
+  const auto& y_div_alpha = to_ref(y_val * inv_alpha);
+  const auto& y_div_alpha_pow_beta
+      = to_ref_if<!is_constant_all<T_shape>::value>(pow(y_div_alpha, beta_val));
+  const auto& log1_arg
+      = to_ref_if<!is_constant_all<T_y, T_scale, T_shape>::value>(
+          1 + y_div_alpha_pow_beta);
+  const auto& log_y = to_ref_if<!is_constant_all<T_shape>::value>(log(y_val));
+  const auto& log_alpha
+      = to_ref_if<include_summand<propto, T_scale, T_shape>::value>(
+          log(alpha_val));
+  const auto& beta_minus_one
+      = to_ref_if < include_summand<propto, T_scale, T_shape>::value
+        || !is_constant_all<T_y>::value > (beta_val - 1.0);
```

Comment:
This is correct, just autoformatter is misparsing it. This will make the code a bit more readable.
```suggestion
      = to_ref_if<(include_summand<propto, T_scale, T_shape>::value
        || !is_constant_all<T_y>::value)>(beta_val - 1.0);
```